fe/fe-core/src/main/java/org/apache/doris/nereids/rules/RuleType.java

```diff
@@ -460,6 +460,7 @@ public enum RuleType {
 
     // sentinel, use to count rules
     SENTINEL(RuleTypeClass.SENTINEL),
+    SHOR_CIRCUIT_POINT_QUERY(RuleTypeClass.REWRITE),
```

Comment:
add it in rewrite group, such as after L346 before L347

Reply:
done